policy-controller/k8s/index/src/outbound/index/metrics.rs

```diff
@@ -33,7 +33,7 @@ impl Collector for Instrumented {
             None,
             MetricType::Gauge,
         )?;
-        let service_infos = ConstGauge::new(this.service_info.len() as u32);
```

Comment:
Will come back and address metrics correctness in a follow-up if that is a non-blocker for now.